.github/workflows/pr-ci.yaml

```diff
@@ -295,3 +295,56 @@ jobs:
             repo: context.repo.repo,
             body: commentBody
           });
+
+  auto-commit-schemas:
+    needs: [compare-graphql-schema, compare-openapi-schema, checkout-branch, update-schemas]
+    runs-on: ubuntu-latest
+    if: needs.update-schemas.outputs.OPENAPI_STATUS != '0' || needs.update-schemas.outputs.GRAPHQL_STATUS != '0'
+    steps:
+      - name: Check out PR branch
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ needs.checkout-branch.outputs.PR_repo }}
+          ref: ${{ needs.checkout-branch.outputs.PR_head_ref }}
+          submodules: true
+          fetch-depth: 0
+      - name: Set up Bot user
```

Comment:
Better to do it like this, I think:

https://github.com/cryostatio/cryostat-web/blob/7c0824c6a19515823d901c3a894441b9f4860fd9/.github/workflows/submodule.yaml#L32

this way the existing signing key for a CI bot account is used and these commits will be verified.